components/private_channel/browser/static_values.h

```diff
@@ -0,0 +1,26 @@
+  /* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
please rename to constants.h and use `extern const char[]` or `constexpr`

Reply:
- [ ] rename to `contants.h` and use idiomatic consts